_posts/2019-06-27-quarkus-0-18-released.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: 'Quarkus 0.18.0 released'
+date: 2019-06-27
+tags: quarkus release
+author: gsmet
+---
+
+Quarkus 0.18.0 was just released and is now available on Maven Central.
+
+We usually release Quarkus every two weeks and it has just been a week since our last release so why release 0.18.0 so soon?
+Well, we have some good news to share: Quarkus now supports GraalVM 19 (requires it, even) and we wanted to put this version in your hands as quickly as possible.
+
+== What's new?
+
+Here are the main highlights of this release.
+
+=== GraalVM 19 Support
+
+When Oracle released GraalVM 19, it came with some breaking changes over the last RC.
+They also made some fundamental changes to the default behavior of GraalVM, for instance by not initializing all the classes by default at compile time.
+
+We had some adjustments to make on Quarkus but we also discovered some blocking issues in GraalVM itself.
+The Quarkus team worked together with the GraalVM team to get those fixed.
+
+With the release of GraalVM 19.0.2, you can now use Quarkus with GraalVM 19. Well, you have to, actually.
+
+Two important points:
+
+ * You need GraalVM 19.0.2+
+ * The `native-image` executable is not bundled in the GraalVM distribution anymore. You have to install it manually using `$GRAALVM_HOME/bin/gu install native-image`.
+
+=== New Kubernetes Client Extension
+
+We added a Kubernetes client extension, based on the Fabric8 Kubernetes Client, to our extension portfolio.
+
+Kubernetes Operators are quickly emerging as a new class of Cloud Native applications.
+These applications essentially watch the Kubernetes API and react to changes on various resources and can be used to manage the lifecycle of all kinds of complex systems like databases, messaging systems and much much more.
+On the implementation side, Operators benefit from having a very low footprint as there can be potentially a very large number of them running in a cluster.
+
+Until now, the only way to create such low footprint operators was to use the Kubernetes Go client which unnecessarily raised the bar for Java developers.
+This new extension allows developers to code in their environment of choice using the well established Fabric8 Kubernetes Client while also being able to create a low footprint native binary.
+
+=== Gradle Support Improvements
+
+A new contributor, https://github.com/garyttierney[Gary Tierney], is working on improving our Gradle support and has already made some significant improvements for this release.
+
+If you have some issues using Gradle with Quarkus, please open GitHub issues so we can improve the Gradle experience.
+
+=== Full changelog
```

Comment:
=== Contributors

Quarkus has now https://github.com/quarkusio/quarkus/graphs/contributors[113 contributors].
Many many thanks to each and everyone of them.
In particular for this release, thanks to [add list of contributors, can be RHT and ext or only ext, up to you].